prow/github/hmac.go

```diff
@@ -94,31 +95,29 @@ func extractHmacs(repo string, tokenGenerator func() []byte) ([][]byte, error) {
 		// whole file is a single line hmac token.
 		//TODO : Once this code has been released and file has been moved to new format,
 		// we should delete this code and return error.
-		logrus.WithError(err).Info("couldn't unmarshal the hmac secret as hierarchical file. Parsing as single token format")
+		logrus.WithError(err).Trace("Couldn't unmarshal the hmac secret as hierarchical file. Parsing as single token format")
```

Comment:
How often is `yaml.Unmarshal()` called? It should be at most once per 5m, not every time `ValidatePayload` is called. Will you please prioritize fixing this? Fine to do it in a subsequent PR. See how prow/config/secret package for various ways we do this sort of thing.

Reply:
It will be pretty often as it will happen on every webhook event.
Sure, I will submit a followup PR to fix it ASAP.